deploy/docker-swarm/multiple-container/docker-compose.yml

```diff
@@ -140,7 +140,7 @@ services:
 
   reports-etl:
     container_name: reports_etl_container
-    image: us-docker.pkg.dev/boldreports/images/bold-etl:6.1.34
+    image: us-docker.pkg.dev/boldreports/images/bold-etl:6.1.45
```

Comment:
Check and include the release folder name for etl image

Reply:
Modified the folder name.